### PR TITLE
Add `FiatAPI` methods to Liquid SDK

### LINKED ISSUE #262: Add support for Fiat Currency & exchange rates

### PR DESCRIPTION
**Pull request by ok300**
Closes #262

### CODE REVIEW

**Review by roeierez — APPROVED** [review]
LGTM

File: lib/core/src/sdk.rs [hunk 1]
```diff
@@ -109,6 +110,8 @@ impl LiquidSdk {
             bitcoin_chain_service.clone(),
         )?;
 
+        let breez_server = BreezServer::new("https://bs1.breez.technology:443".into(), None)?;
```
Comment (erdemyerebasmaz) [inline lib/core/src/sdk.rs:113]: Looking at the implementation on Breez SDK, should Breez server URL be part of `Config`? 
If applicable here, Production & Staging(Mainnet & Testnet) could use different URLs as part of default config.

**Review by erdemyerebasmaz — APPROVED** [review]
Looks good. Have a single question regarding Breez server url.

File: lib/core/src/sdk.rs (hunk 1 above)
Comment (ok300) [reply lib/core/src/sdk.rs:113]: Good question.

The main difference I see is Breez SDK has Production / Staging configs, whereas Liquid SDK has Mainnet / Testnet configs.

So at the moment I don't see a place for the Staging breez server URL, unless we add e.g. Mainnet Staging.

@roeierez what do you think?

Comment (roeierez) [reply lib/core/src/sdk.rs:113]: staging is no longer exist in breez_sdk (we should actually remove it) soI think we don't need it at the config.
We can add a  `new_mainnet` function in BreezServer sdk-common implementation to return an instance already populated with the mainnet URL so we don't have to pass it from the lqiuid sdk.

Comment (ok300) [reply lib/core/src/sdk.rs:114]: The Breez Server URLs are defined in static variables in Breez SDK.

I moved them to `sdk-common` so they can easily be referenced here: https://github.com/breez/breez-sdk/pull/1023 . I can then reference `PRODUCTION_BREEZSERVER_URL` here.

Would this be ok as a solution?

Comment (roeierez) [reply lib/core/src/sdk.rs:114]: IMO yes.

Comment (ok300) [reply lib/core/src/sdk.rs]: Done: 71647e33753248169479d3b731a8ec703fb5613e

**Pull request merged**